docs/architecture.md

```diff
@@ -53,22 +53,18 @@ The participant network is the beating heart at the center of the package. The p
 
 We'll explain these phases one by one.
 
-### Generating EL client data
+### Generating EL and CL client data
 
-All EL clients require both a genesis file and a JWT secret. The exact format of the genesis file differs per client, so we first leverage [a Docker image containing tools for generating this genesis data][ethereum-genesis-generator] to create the actual files that the EL clients-to-be will need. This is accomplished by filling in several genesis generation config files found in [the `static_files` directory][static-files].
```

Comment:
lets get rid of the anchor down below if we aren't using it

Reply:
which anchor do you mean?